public/locales/ar/about.json

```diff
@@ -0,0 +1,7 @@
+{
+    "HEALING!": "شفاء!",
+    "Bringing hope through therapy.": "نزرع الأمل من العلاج.",
+    "introAboutPage": "في Healing ، نعتقد أن هناك طريقة ي Healing ، نعتقد أن هناك طريقة أفضل للقيام بالأشياء. طريقة أكثر قيمة لكسب العملاء ، بدلاً من شرائهم. نحن مهووسون بشغف بهذه المهمة ، وهي مساعدة الناس على تحقيق النجاح. ينصب تركيزنا على تحسين محرك البحث ، والذي غالبًا ما يُساء فهمه ويكون مبهمًا. نرى هذا كفرصة لتبسيطها للجميع من خلال برامجنا وتعليمنا ومجت أفضل للقيام بالأشياء. طريقة أكثر قيمة لكسب العملاء ، بدلاً من شرائهم. نحن مهووسون بشغف بهذه المهمة ، وهي مساعدة الناس على تحقيق النجاح. ينصب تركيزنا على تحسين محرك البحث ، والذي غالبًا ما يُساء فهمه ويكون مبهمًا. نرى هذا كفرصة لتبسيطها للجميع من خلال برامجنا وتعليمنا ومجتمعنا",
+    "Our Founding": "الإنطلاق",
+    "AboutHealing": "تم تأسيس Healing بواسطة بيان أبوبكر في 2020. كان يطلق عليه Healing Online ، وقد بدأ كمدونة ومجتمع عبر الإنترنت حيث شارك بعض خبراء المسرح في العالم بأبحاثهم وأفكارهم. لقد أنشأنا دليل المبتدئين إلى عمل والدراسة الأولى ، وتحولت خبرة الصناعة إلى استشارة صغيرة وقادتنا إلى إنشاء المعالج عبر الإنترنت اليوم!"
```

Comment:
- I think it will be better to make `keys` ad `camelCase`:
`Our Founding` can be `ourFounding`, `AboutHealing` can be `aboutHealing`